### ISSUE TITLE
Tweak Content.slug to ignore links

### ISSUE THREAD

**Issue by jaywink** [jaywink/socialhome#245]
We use the slug in urls and content that starts with an embedded image makes silly slugs.

Change `Content.slug` to remove all links from the text before creating a short text + slug from it.

**Comment by takumab** [comment]
This sounds great can I work on this issue?

**Comment by jaywink** [comment]
@takumab sure, feel free to! If you have any questions, you can ask here or we have a [chatroom](https://socialhome.readthedocs.io/en/latest/community.html#feedback-and-community-chat). Don't be afraid to ask if things come up!

**Comment by takumab** [comment]
Thanks. This is the strange slug I need to remove correct? 

![screen shot 2017-10-20 at 8 53 51 pm](https://user-images.githubusercontent.com/14036745/31839486-311da9b2-b5d9-11e7-899f-308e9cae9268.png)

**Comment by jaywink** [comment]
Yes. Also, even worse, when the url is before any text, the slug can be something like this:

    httpspodorkznetuploadsimagese5ee53ce55/

I think a good way of approaching is to write a test case with for example the following content:

    https://example.com/somerandomlongstringastheurlpart/
    
    This is the actual body text we're interested to get into the slug.

Then make the test assert that the slug contains only the relevant part of the body text. We have a pretty decent url finding function [here](https://github.com/jaywink/socialhome/blob/5cd0026a3692982d753789055bd83b853673a9c0/socialhome/content/utils.py#L101) you could use to find and remove the urls, for example. Actually, the bleach library we use can also [remove them](https://bleach.readthedocs.io/en/latest/linkify.html#removing-links) - this might be the way to go imho. An empty slug is fine, if `Content.text` really only contains a link in one form or another.

**Comment by takumab** [comment]
Hi @jaywink, 
Hope you are well. I am getting this error: 
`ValueError: Unable to configure handler 'syslog': [Errno 2] No such file or directory` 
Wondering if I am doing something wrong. Thanks.

**Comment by takumab** [comment]
Oh that error occurs whenever I try to run test

**Comment by jaywink** [comment]
Which OS are you running? I think there might be a problem in the configuration, when I added syslog logging possibility. I think I've fixed it now for non-Linux environments - if this is the problem.

I just pushed [this commit](https://github.com/jaywink/socialhome/commit/4c1b068e287d38fb7d4ca948edca75303f71fe7e) to master which I think will fix this situation. Does it work for you once you update from master?

**Comment by takumab** [comment]
@jaywink I am using Mac OS X. I will check later on tonight when I get off work. 
Thanks
